docs/Makefile

```diff
@@ -24,7 +24,7 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html"
 
 livehtml:
-	sphinx-autobuild -p $(PORT) -b html $(SOURCEDIR) $(BUILDDIR)/html $(O)
+	sphinx-autobuild --port $(PORT) -b html $(SOURCEDIR) $(BUILDDIR)/html $(O) --open-browser
```

Comment:
Didn't know about this, thanks for updating this.

Reply:
happy it helped :)